.github/workflows/integration-tests.yml

```diff
@@ -4,20 +4,22 @@
 
 name: Integration tests
 
-on:
-  schedule:
-    - cron:  '0 0 * * 4'
+on: pull_request
+  #schedule:
+  #  - cron:  '0 0 * * 4'
 
 jobs:
   test:
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        # Note that Ubuntu 20 cannot be used because its SWIG version is 4.0
+        # (enthought/enable#360)
```

Comment:
i'd be surprised if we can't explicitly install swig 3 on ubuntu 20

Reply:
I've opened #1660